docs/developer-guide/ksqldb-reference/select-pull-query.md

```diff
@@ -36,12 +36,40 @@ Execute a pull query by sending an HTTP request to the ksqlDB REST API, and
 the API responds with a single response.  
 
 The WHERE clause must contain a value for each primary-key column to retrieve and may
-optionally include bounds on `WINDOWSTART` and `WINDOWEND` if the materialized table is windowed.
+optionally include bounds on `WINDOWSTART` and `WINDOWEND` if the materialized view is windowed.

```

Comment:
We should clarify this in more detail for when table scans are enabled.  We should say, after the "Time and WIndows" link, that enabling table scans removes many of the restrictions and allows most queries, though it'll be less performant, depending on the size of the underlying view.